src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -1175,4 +1191,171 @@ mod tests {
         );
         assert_eq!(unsafe { CONFIRM_COUNTER }, 4);
     }
+
+    /// Test that the chain confirmation screen appears for known non-mainnet networks.
+    #[test]
+    pub fn test_chain_confirmation_for_l2_networks() {
```

Comment:
is it fair to call all of them l2 (or sidechains)? Aren't some normal base-layer chains in their own right?

Reply:
sure, can change the name